lib/github-pages/configuration.rb

```diff
@@ -162,15 +164,6 @@ def debug_print_versions
         Jekyll.logger.debug "GitHub Pages:", "github-pages v#{GitHubPages::VERSION}"
         Jekyll.logger.debug "GitHub Pages:", "jekyll v#{Jekyll::VERSION}"
       end
-
-      # To minimize erorrs, lazy-load plugins if they're requested by the user
-      def conditionally_require_plugins(site)
-        plugins = []
-        plugins.push "jekyll-remote-theme" if site.config.key? "remote_theme"
-        Jekyll::External.require_with_graceful_fail(
-          plugins.select { |plugin| site.plugin_manager.plugin_allowed?(plugin) }
```

Comment:
Is it ok to remove this? This will change behaviour.

Reply:
> This will change behaviour.

How so? It will be in the config, as if the user had explicitly added it (or it was a default plugin, as it was before), instead of being manually required after the site's config is built.

Reply:
Totally right. Don't mind me 🎶